toolchain/src/integration-test/resources/input.ttl

```diff
@@ -0,0 +1,973 @@
+@prefix :      <http://arch-ont.org/ontologies/architectureconformance#> .
```

Comment:
Should this file be committed at all?

Reply:
Uh, no. I will fix it.

Reply:
:+1:
If commits get squashed before completion of PR, this file will never appear in history.